app/assets/javascripts/discourse/tests/fixtures/topic.js

```diff
@@ -2734,14 +2734,14 @@ export default {
           domain: "github.com",
         },
         {
-          url: "http://pootle.locamotion.org/",
+          url: "http://pootle.locomotion.org/",
```

Comment:
I'm going to drop this. Apparently it's a domain (although it isn't responding to http/https requests)